luaui/Widgets/gui_healthbars_gl4.lua

```diff
@@ -1009,6 +1017,8 @@ end
 local uniformcache = {0.0}
 
 local function addBarsForUnit(unitID, unitDefID, unitTeam, unitAllyTeam, reason) -- TODO, actually, we need to check for all of these for stuff entering LOS
+	ui_scale = tonumber(Spring.GetConfigFloat("ui_scale", 1) or 1)
```

Comment:
This value is reinitialized with uiscale setting is changed and is synchronzed with the other callins, unfortunately due to the fact the whole widget is reinitialized instead of just being called for reinit.